scripts/metric.py

```diff
@@ -57,12 +71,12 @@
 compiler = Ncompiler
 runtime = Nruntime
 races = Nraces
-for i in range(1,len(Ntruth)):
+for i in range(len(Ntruth)):
 	if compiler[i] == '0':
 		compilertrue += 1
 		if runtime[i] == '0':
 			runtimetrue += 1
-			if truth[i] == 'true':
+			if truth[i] == 'TRUE':
```

Comment:
In the csv file I'm looking at, true is lower case. I suggest:
```
if truth[i].upper() == 'TRUE':
```
Even if newly generated files might have TRUE now, this would allow to process existing files.

Reply:
I just incorporated your suggestion.  Thanks again!